pkg/controllers/dynakube/dynatraceclient/builder_test.go

```diff
@@ -39,13 +40,13 @@ func TestBuildDynatraceClient(t *testing.T) {
 			},
 			dk: *dk,
 		}
-		dtc, err := dynatraceClientBuilder.Build()
+		dtc, err := dynatraceClientBuilder.Build(context.Background())
```

Comment:
small nitpick:

you could just define the ctx in the beginning

```
func TestBuildDynatraceClient(t *testing.T) {
	ctx := context.Background()
...
}

Reply:
right
[9c177cc](https://github.com/Dynatrace/dynatrace-operator/pull/5119/commits/9c177cc910b8e6fbe7405ae3271214c05440944b)